gnd/src/main/java/com/google/android/gnd/persistence/local/LocalValueStore.java

```diff
@@ -72,4 +73,19 @@ public void saveMapType(int type) {
   public int getSavedMapType(int defaultType) {
     return preferences.getInt(MAP_TYPE, defaultType);
   }
+
+  /**
+   * This method will return whether the user have accepted the terms or not.
+   * @return true or false depending upon the user.
+   */
+  public boolean areTermsAccepted() {
+    return preferences.getBoolean(TERMS_ACCEPTED, false);
+  }
+
+  /**
+   * This method will update terms_accepted value.
```

Comment:
This is clear from the method description, so can omit. Instead, may be worth mentioning this related to the currently signed in user, e.g.

"Updates the terms of service acceptance state for the currently signed in user."

Reply:
updated in https://github.com/google/ground-android/pull/845